vm/chef/cookbooks/espocrm/files/espocrm-utils

```diff
@@ -81,19 +81,20 @@ function change_php_config() {
 #######################################
 function install_app() {
   local -r curl="curl --request POST -c /tmp/cookie -b /tmp/cookie"
+  local -r URL="http://localhost/install/"
```

Comment:
as a local variable, please rename `URL` to `url`

https://google.github.io/styleguide/shellguide.html#use-local-variables

Reply:
Done